src/content/en/fundamentals/media/manipulating/files.md

```diff
@@ -0,0 +1,325 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: TBD.
+
+{# wf_updated_on: 2017-06-23 #}
+{# wf_published_on: 2017-06-23 #}
+
+# Media File Manipulation {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+Now that I've introduced you to applications that can be used for media file
+manipulation, I'm going to take a raw video file off a camera and transform it
+into an encrypted resource that you can play back using a video library such as
+[Google's Shaka Player](https://shaka-player-demo.appspot.com/demo/). I'm
+specifically going to show you how to format your video for mobile playback.
+
+Objective: This article provides explanations of file manipulation concepts,
+with command lines only to illustrate the concepts. There is a companion [cheat
+sheet](cheatsheet) that shows more commands and is designed as a quick reference
+for someone who knows the concepts.
+
+To be more specific about the goals, the result of the procedures described here
+will be media resources with the following characteristics:
+
++  Audio and video streams are split into separate files
```

Comment:
See line 106. I said it's done that way for HLS and DASH. I guess I should have asked if HLS and DASH required this. (I thought it did, but maybe it doesn't.) Even if it doesn't, it would seem to me to be more efficient if the dynamic part of DASH only had to deal with one _type_ of stream at a time. 

Maybe @johnpallett or @beaufortfrancois could comment on this.